oak-segment-azure/src/main/java/org/apache/jackrabbit/oak/segment/azure/AzureUtilities.java

```diff
@@ -116,6 +116,15 @@ public static CloudBlobDirectory cloudBlobDirectoryFrom(String connection, Strin
 
         return container.getDirectoryReference(dir);
     }
+    public static CloudBlobDirectory cloudBlobDirectoryFrom(String connection, String containerName,
```

Comment:
I believe this method is not necessary. Rather previous signature could be changed, and add **skipCreateContainer** parameter. Seems it is only used in org.apache.jackrabbit.oak.run.DataStoreCommandTest.StoreFixture.AzureSegmentStoreFixture#init

Reply:
@smiroslav  It's me that suggested to add an overload of `cloudBlobDirectoryFrom()`, in order to keep backward compatibility in case 3rd-party code is using it. Do you think it's not necessary?

If we keep them, however, one overload should call the second, instead of re-implementing it:
```java
public static CloudBlobDirectory cloudBlobDirectoryFrom(String connection, String containerName,
            String dir) throws InvalidKeyException, URISyntaxException, StorageException {
  return cloudBlobDirectoryFrom(connection, containerName, dir, false);
}
```

Reply:
@jelmini you are right. Util classes are also part of the public API and we should maintain compatibility. 
Your suggested approach can be used to prevent code duplication.

Reply:
@ArunOnCloud I think this is still not resolved. to prevent code duplication, the suggestion from @jelmini should be applied.

Reply:
@smiroslav Yes I missed re-implementation part.  I will resolve it in next commit, if everything else is ok.